### DISCUSSION TITLE
About comment plugins

### DISCUSSION THREAD

**Discussion by CuteReimu** [vuepress-theme-hope/vuepress-theme-hope#4660] [Q&A] [answered]
It says in vuepress-theme-hope document that we should install comment plugins ourselves by:

```shell
npm i -D @waline/client
```

But I encountered an error in production environment:

```
Failed to load resource: the server responded with a status of 404 (Not Found)
                                              assets/waline-meta-l0sNRNKZ.js:1 
```

But everything runs well. And I can use the comment plugins as well.

Should we install the plugins via:

```shell
npm i @waline/client
```

Because I think we may use this plugin in our client?

**Comment by Mister-Hope** [comment, marked as answer]
It's an edge case that cause an upstream bug

https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/4496

The chunk has only css and is extracted to a bundled css, leaving the chunk empty, so vite no longer generates it.
